api/base/serializers.py

```diff
@@ -34,6 +36,59 @@ def _url_val(val, obj, serializer, **kwargs):
         return val
 
 
+class HyperlinkedIdentityFieldWithMeta(ser.HyperlinkedIdentityField):
```

Comment:
Perhaps to establish this as the canonical JSON API hyperlinking field and reflect the fact that this field does JSON API-specific formatting, we rename this to `JSONAPIHyperlinkedIdentityField`.

Plus, it's a whole 1 character shorter!